package/gluon-mesh-vpn-tunneldigger/files/usr/bin/tunneldigger-watchdog

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/lua
+
+function restartTunneldigger()
+	io.popen('logger -t tunneldigger-watchdog "Restarting Tunneldigger."')
+	os.execute("/etc/init.d/tunneldigger restart")
+end
+
+function readPidFile()
```

Comment:
I would turn the logic in this function around like this

```lua
function read_pid_file()
	local file = io.open("/var/run/tunneldigger.mesh-vpn.pid", "r")
	if not file:
		return nil
	pid = file:read("*l")
	file:close()
	return pid
end
```

Reply:
Strongly disagree. But we're not the maintainers :).

Reply:
I'm not a lua fanatic either, so I don't have a strong opinion about this.